solidity/test/Converter.js

```diff
@@ -114,7 +114,7 @@ contract('Converter', accounts => {
 
     const createAnchor = async (type) => {
         switch (type) {
-            case 0: 
+            case 0:
                 anchor = await SmartToken.new('Token1', 'TKN1', 2);
```

Comment:
Don't we want `18` here too?

Reply:
Yeah, it'd be better to test 18 for the sake of the tests

Reply:
So it's currently `2`.

Reply:
We only use larger amounts in V2 tests, which is where it was fixed. Everywhere else is exactly as it was before